packages/eccodes/package.py

```diff
@@ -107,6 +107,9 @@ def cmake_args(self):
         if self.spec.variants['jp2k'].value == 'openjpeg':
             args.append('-DOPENJPEG_PATH=' + self.spec['openjpeg'].prefix)
 
+        if self.spec.variants['jp2k'].value == 'jasper':
+            args.append('-DJASPER_PATH=' + self.spec['jasper'].prefix)
```

Comment:
and if variant is not jasper, does it work? dont we need to add equivalent for openjpeg ?